recipes-extended/images/tb-xen-image.bb

```diff
@@ -0,0 +1,14 @@
+# nooelint: oelint.file.requirenotfound
+require recipes-extended/images/xen-image-minimal.bb
+
+SUMMARY = "A small XEN image just capable of allowing a device to boot."
+DESCRIPTION = "A small XEN image just capable of allowing a device to boot."
+LICENSE = "MIT"
+
```

Comment:
Addressed in a6bb6bf77442194d5733cff4386bf6d81542ae4e

I decided not to touch the original (non-xen) recipe file, since it is sort of out of scope of this PR.